apps/web/src/components/HolidayAlert/HolidayAlert.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { useTheme } from "@gc-digital-talent/theme";
+import { AlertImage } from "@gc-digital-talent/ui";
+
+import lightImage from "~/assets/img/Holiday_2023_graphic_Light_mode.webp";
+import darkImage from "~/assets/img/Holiday_2023_graphic_darkmode.webp";
+
+const HolidayAlert = (): JSX.Element | null => {
+  const { mode } = useTheme();
+  const intl = useIntl();
+
+  const dateNow = new Date();
+  const endOfAlertTime = new Date("2024-01-08 12:00:00");
+  const holidayAlertActive = dateNow.getTime() < endOfAlertTime.getTime();
+
+  const message = (
+    <div
+      data-h2-justify-content="base(space-between)"
+      data-h2-max-height="base(100%)"
+    >
+      <h2>
+        {intl.formatMessage({
+          defaultMessage: "Happy Holidays!",
+          id: "tQseOY",
+          description: "Happy holidays, seasons greetings heading",
+        })}
+      </h2>
```

Comment:
Could we use the `Heading` component here?

```tsx
<Heading size="h6" data-h2-margin-top="base(0)">
```

Reply:
Thanks for the suggestions! Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8769/commits/66dd2de47dccadbc1d14daa884ee260451c46460